project_event/models/project_task.py

```diff
@@ -102,6 +102,9 @@ class Task(models.Model):
         default='draft',
         track_visibility='onchange',
     )
+    reservation_event_id = fields.Integer(
```

Comment:
@eilst : required=True, why name is required fields. If Yes why name field in project.project is not set required . Please apply the same comments where applicable